From: Andrea Bolognani <eof@kiyuko.org>
Date: Sat, 7 Dec 2019 16:00:08 +0100
Subject: Fix flake8 warnings

The only remaining issues reported by the tool are

  rhinote.py:59:80: E501 line too long (81 > 79 characters)
  rhinote.py:69:80: E501 line too long (83 > 79 characters)

but I think addressing them would actually make the code less
readable, so I decided not to.
---
 rhinote.py | 35 +++++++++++++++++++----------------
 1 file changed, 19 insertions(+), 16 deletions(-)

diff --git a/rhinote.py b/rhinote.py
index b56f938..93e2363 100644
--- a/rhinote.py
+++ b/rhinote.py
@@ -14,7 +14,7 @@
 
 # This program is distributed in hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 # GNU General Public License for more details.
 
 # You should have received a copy of the GNU General Public License
@@ -26,21 +26,23 @@ import tkinter.filedialog as tkFileDialog
 import tkinter.messagebox as tkMessageBox
 import os
 
+
 # the root window:
 def Rhinote():
     r = tkinter.Tk()
     r.option_add('*font', '{Helvetica} 11')
-    t = TextWidget(r, bg = '#f9f3a9', wrap = 'word', undo = True)
+    t = TextWidget(r, bg='#f9f3a9', wrap='word', undo=True)
     t.focus_set()
-    t.pack(fill = 'both', expand = 1)
+    t.pack(fill='both', expand=1)
     r.geometry('220x235')
     r.title('Rhinote')
     r.mainloop()
 
+
 # the text widget, and all of its functions:
 class TextWidget(tkinter.Text):
 
-    def save_file(self, whatever = None):
+    def save_file(self, whatever=None):
         if (self.filename == ''):
             self.save_file_as()
             self.master.title('Rhinote %s' % self.filename)
@@ -49,12 +51,12 @@ class TextWidget(tkinter.Text):
             f.write(self.get('1.0', 'end'))
             f.close()
             self.master.title('Rhinote %s' % self.filename)
-            # Comment out the following line if you don't want a 
+            # Comment out the following line if you don't want a
             # pop-up message every time you save a file:
             tkMessageBox.showinfo('FYI', 'File Saved.')
 
-    def save_file_as(self, whatever = None):
-        self.filename = tkFileDialog.asksaveasfilename(filetypes = self._filetypes)
+    def save_file_as(self, whatever=None):
+        self.filename = tkFileDialog.asksaveasfilename(filetypes=self._filetypes)
         f = open(self.filename, 'w')
         f.write(self.get('1.0', 'end'))
         f.close()
@@ -62,9 +64,9 @@ class TextWidget(tkinter.Text):
         # pop-up message every time you save a file:
         tkMessageBox.showinfo('FYI', 'File Saved')
 
-    def open_file(self, whatever = None, filename = None):
+    def open_file(self, whatever=None, filename=None):
         if not filename:
-            self.filename = tkFileDialog.askopenfilename(filetypes = self._filetypes)
+            self.filename = tkFileDialog.askopenfilename(filetypes=self._filetypes)
             self.master.title('Rhinote %s' % self.filename)
         else:
             self.filename = filename
@@ -80,7 +82,7 @@ class TextWidget(tkinter.Text):
     def new_window(self, event):
         Rhinote()
 
-    def printfile(self, whatever = None):
+    def printfile(self, whatever=None):
         f = open(self.printfilename, 'w')
         f.write(self.get('1.0', 'end'))
         f.close
@@ -88,8 +90,8 @@ class TextWidget(tkinter.Text):
         # enscript's -B option suppresses page headers.
         os.system('enscript -B --word-wrap $HOME/.Rhinoteprintfile > lpr &')
 
-    def help(self, whatever = None):
-        tkMessageBox.showinfo('Rhinote Help', message = '''
+    def help(self, whatever=None):
+        tkMessageBox.showinfo('Rhinote Help', message='''
 Editing Commands
     Ctrl-x : Cut selected text
     Ctrl-c : Copy selected text
@@ -128,12 +130,13 @@ http://rhinote.tuxfamily.org
         self.bind('<Control-H>', self.help)
         self.master = master
         self.filename = ''
-        self.printfilename = os.environ['HOME']+'/.Rhinoteprintfile'
+        self.printfilename = os.environ['HOME'] + '/.Rhinoteprintfile'
         self._filetypes = [
-        ('Text/ASCII', '*.txt'),
-        ('Rhinote files', '*.rhi'),
+            ('Text/ASCII', '*.txt'),
+            ('Rhinote files', '*.rhi'),
             ('All files', '*'),
-            ]
+        ]
+
 
 # make it so:
 if __name__ == '__main__':
